src/backend/src/oisy_user.rs

```diff
@@ -0,0 +1,40 @@
+use crate::{Candid, StoredPrincipal, VMem};
+use candid::Principal;
+use ic_stable_structures::StableBTreeMap;
+use shared::types::{
+    user_profile::{ListUsersRequest, OisyUser, StoredUserProfile},
+    Timestamp,
+};
+use std::ops::Bound;
+
+const DEFAULT_LIMIT_LIST_USERS_RESPONSE: usize = 10_000;
+const PRINCIPAL_MIN: Principal = Principal::from_slice(&[]);
+
```

Comment:
Note: The `get_` prefix is normally not used according to the Rust naming guidelines: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

The same applies to various other getters below.